tobac/tests/test_import.py

```diff
@@ -4,3 +4,10 @@
 
 def test_dummy_function():
     assert 1 == 1
+
+
+def test_version():
+    """Test to make sure that we have a version number included.
+    If it's not, this should result in an error.
+    """
+    assert tobac.__version__ == tobac.__version__
```

Comment:
Note that we do not test the `get_version()` function in `setup.py` but only if we have defined `__version__` in `__init__.py` but I guess that is fine

Reply:
I'm not sure that we can test functions in setup.py? We also can't move these functions inside the library, because it isn't yet installed.

Integration testing is still performed in our CI workflow though, as we `pip install . ` in our Dockerfile.

Reply:
You are right and good point with the Docker image - actually the testing failed when you forgot to update one line in your last commit (see my comment below), which is great so we know this is covered in the workflow :)